### PR TITLE
Fixed API and CLI test cases failure

### PR DESCRIPTION
**Pull request by devendra104**
Fixed Existence Test Failure related to API and CLI.

### CODE REVIEW

**Comment by devendra104** [comment]
API Test Results:

$ pytest -s upgrade_tests/test_existance_relations/api/test_domains.py 
upgrade_tests/test_existance_relations/api/test_domains.py ........('Sorted module is not supported in python3 for dict to dict comparison and '
 "gets error like: '<' not supported between instances of 'dict' and 'dict'")
====================================================================================== 9 passed, 2 warnings in 0.29 seconds =======================================================================================

$ pytest -s upgrade_tests/test_existance_relations/cli/test_filter.py 
================================================================================ 5 failed, 2475 passed, 2 warnings in 4.64 seconds ================================================================================

etc.

File: upgrade_tests/helpers/common.py [hunk 1]
```diff
@@ -68,7 +69,13 @@ def test_anyentity(pre, post)
         return assert_templates(template, pre, post)
     else:
         if isinstance(pre, list) and isinstance(post, list):
-            return sorted(list(pre)) == sorted(list(post))
+            try:
+                return sorted(list(pre)) == sorted(list(post))
+            except TypeError as err:
+                pprint('Sorted module is not supported in python3 for dict to dict '
+                       'comparison and gets error like: {}'.format(err))
+                return pre.sort(key=lambda item: item.get("id")) == \
```
Comment (jyejare) [inline upgrade_tests/helpers/common.py]: Does this works in case pre / post is type of list ?

Comment (devendra104) [reply upgrade_tests/helpers/common.py]: Yes, it worked.

Comment (devendra104) [reply upgrade_tests/helpers/common.py]: I have shared the test result in my previous comment.

**Pull request merged**